framework/test/integration/common.js

```diff
@@ -237,15 +237,16 @@ function getAccountFromDb(library, address) {
 		library.storage.adapter.execute(
 			`SELECT * FROM mem_accounts2multisignatures where "accountId" = '${address}'`
 		),
-		library.storage.adapter.db.query(
-			`SELECT * FROM mem_accounts2u_multisignatures where "accountId" = '${address}'`
-		),
+		// [UNCONFIRMED_STATE_REMOVAL]
```

Comment:
Can you change the documentation here and in all of the occurrences to refer to the issue, which will fix these? 
Sth like
"// [UNCONFIRMED_STATE_REMOVAL] - Uncomment as part of https://github.com/LiskHQ/lisk/issues/2918`

Reply:
Will take care of these comments once we retake merging of this PR as it's suspended now as we figured it could be better to try to merge once the rest of the feature is ready.